am-do-2-atom-do.py

```diff
@@ -0,0 +1,410 @@
+            mysqlCursor.execute(sql, (file['object_id'], object_uuid['value'], aip_uuid['value'], False))
+            mysqlConnection.commit()
+        except Exception as e:
+            print("Unable to insert working data for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+        try:
+            # Flush the existing properties for the legacy digital file.
+            # Includes an automatic cascade delete of the i18n value.
+            # These properties will be replaced with values from the METS file.
+            sql = "DELETE FROM property WHERE object_id = %s;"
+            mysqlCursor.execute(sql, file['object_id'])
+            mysqlConnection.commit()
+        except Exception as e:
+            print("Unable to flush existing property values for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+    return
+
+
+def get_mets_path(aip_uuid):
+    request_url = STORAGE_SERVICE_URL + "file/" + aip_uuid + "?username=" + STORAGE_SERVICE_USER + "&api_key=" + STORAGE_SERVICE_API_KEY
+    try:
+        response = requests.get(request_url)
+    except Exception as e:
+        print(e)
+        sys.exit("Unable to connect to Storage Service. Check your connection parameters.")
+
+    package = response.json()
+
+    # build relative path to METS file
+    if package["current_path"].endswith(".7z"):
+        relativePath = package["current_path"][40:-3]
+    else:
+        relativePath = package["current_path"][40:]
+    relativePathToMETS = (
+        relativePath + "/data/METS." + package["uuid"] + ".xml"
+    )
+
+    # Derive AIP transfer name from filepath value by removing UUID suffix
+    transfer_name = relativePath[:-37]
+
+    return relativePathToMETS, transfer_name
+
+
+def get_mets_file(aip_uuid, relative_path):
+    request_url = STORAGE_SERVICE_URL + "file/" + aip_uuid + "/extract_file/?relative_path_to_file=" + relative_path + "&username=" + STORAGE_SERVICE_USER + "&api_key=" + STORAGE_SERVICE_API_KEY
+    response = requests.get(request_url)
+    mets_file = "METS.{}.xml".format(aip_uuid)
+    download_file = os.path.join(METS_DIR, mets_file)
+    with open(download_file, "wb") as file:
+        file.write(response.content)
+
+
+def parse_mets_values(aip_uuid):
+    global ERROR_COUNT
+
+    # Identify all AtoM digital objects in this Archivematica AIP.
+    try:
+        sql = "SELECT * FROM dip_files WHERE aip_uuid = %s;"
+        mysqlCursor.execute(sql, aip_uuid)
+        legacy_dip_files = mysqlCursor.fetchall()
+    except Exception as e:
+        print("Unable to fetch the digital object records associated with AIP " + aip_uuid)
+        print(e)
+        return
+
+    # Download METS file if a local copy is not present.
+    if os.path.exists(METS_DIR + aip_uuid + ".xml") is False:
```

Comment:
When testing truthiness for e.g. the existence of a path, it's better to use `if not`

```suggestion
    mets_path = os.path.join(METS_DIR, "METS.{}.xml".format(aip_uuid))
    if not os.path.exists(mets_path):